app/models/track/concept.rb

```diff
@@ -17,9 +17,20 @@ def blurb
     "C# structs are closely related to classes. They have state and behavior. They have constructors that take arguments, instances can be assigned, tested for equality and stored in collections." # rubocop:disable Layout/LineLength
   end
 
-  # Cache this
+  memoize
+  def concept_exercises
+    ConceptExercise.that_teach(self)
+  end
+
+  memoize
+  def practice_exercises
+    PracticeExercise.that_practice(self)
+  end
+
+  # TODO: Cache this?
+  memoize
   def num_exercises
-    3
+    concept_exercises.size + practice_exercises.size
```

Comment:
We'll almost certainly end up removing all references to this and using the summary instead.